main/modes/music/tunernome/tunernome.c

```diff
@@ -44,20 +47,37 @@
 #define SENSITIVITY                  5
 #define TONAL_DIFF_IN_TUNE_DEVIATION 10
 
-#define METRONOME_CENTER_X   TFT_WIDTH / 2
-#define METRONOME_CENTER_Y   TFT_HEIGHT - (2 * tunernome->ibm_vga8.height) - 10 - CORNER_OFFSET
-#define TUNER_CENTER_Y       TFT_HEIGHT - tunernome->ibm_vga8.height - 8 - CORNER_OFFSET
-#define METRONOME_RADIUS     125
-#define TUNER_RADIUS         80
-#define TUNER_TEXT_Y_OFFSET  30
-#define TUNER_ARROW_Y_OFFSET 8
-#define INITIAL_BPM          120
-#define MAX_BPM              400
-#define MAX_BEATS            16 // I mean, realistically, who's going to have more than 16 beats in a measure?
-// #define METRONOME_CLICK_MS   35
-#define BPM_CHANGE_FIRST_MS  500
-#define BPM_CHANGE_FAST_MS   2000
-#define BPM_CHANGE_REPEAT_MS 50
+// Tuner screen definitions
+#define TUNER_CENTER_Y                (TFT_HEIGHT - tunernome->ibm_vga8.height - 8 - CORNER_OFFSET)
```

Comment:
I don't _love_ using variables in a macro without args b/c i this case it requires `tunernome` to be in the same scope wherever this macro is used. That happens to be the one global var, so its fine, just be wary in the future.